doc/rest-api.yaml

```diff
@@ -373,7 +373,7 @@ definitions:
         type: string
         x-go-name: Status
       type:
-        description: The network name
+        description: The network type
```

Comment:
This file is auto-generated, instead you should fix `shared/api/network.go` where the typo originates from.

Reply:
Thanks, I amended my commit.